tests/src/Simple/Collision/Collision.il

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+.assembly extern System.Runtime
```

Comment:
Yes but I had issues to include the right version of either mscorlib or System.Runtime using the C# compiler included in the dotnet cli that I wrote this small IL instead.